mne/viz/utils.py

```diff
@@ -38,7 +38,7 @@ def _setup_vmin_vmax(data, vmin, vmax):
             vmin = np.min(data)
         if callable(vmax):
             vmax = vmax(data)
-        elif vmin is None:
+        elif vmax is None:
             vmax = np.max(data)
```

Comment:
Docs from ``plot_tfr_topomap`` say:
> vmin : float | callable
        The value specfying the lower bound of the color range.
        If None, and vmax is None, -vmax is used. Else np.min(data).
        If callable, the output equals vmin(data).
    vmax : float | callable
        The value specfying the upper bound of the color range.
        If None, the maximum absolute value is used. If vmin is None,
        but vmax is not, defaults to np.min(data).
        If callable, the output equals vmax(data).

git blames @agramfort